scripts/contentScript.js

```diff
@@ -380,6 +380,154 @@ async function FilesInPrBitbucket(response) {
 	});
 }
 
+const githubHunkHighlight = async (apiResponses) => {
+
+	// TODO: optimization  needed to not highlight next deleted line space if not present in response.
+	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
+	getFileName.forEach(async (item) => {
+		let fileContent = item.getAttribute('data-tagsearch-path');
+		if (fileContent) {
+
+			const matchEncrypted = await sha256(fileContent);
+			const foundFile = apiResponses.find(item => item.filepath === matchEncrypted);
+
+			if (foundFile) {
+				// checking for Diff view either undefined or split 
+				const deletedLines = document.querySelectorAll('input[value]');
+				let diffView = false;
+				deletedLines.forEach((item) => {
+					const getValue = item.getAttribute('value');
```

Comment:
Interesting... do we have to do this for each diff? Isn't it set at a page level?
I mean, if one diff is split, then all diffs should be split, and if one diff is unified, then all diffs should be unified. Isn't it?